src/blueapi/service/main.py

```diff
@@ -154,6 +157,29 @@ def submit_task(
         ) from e
 
 
+example_query = Query(
+    TaskStatusEnum.UNSTARTED, description="The status of the tasks to retrieve"
+)
+
+
+@app.get("/tasks")
+def get_tasks(
+    status: TaskStatusEnum = example_query,
```

Comment:
I mean make a brand new pydantic model, something like

```python
class TaskRequest(BlueapiBaseModel):
    status: TaskStatusEnum
```

Reply:
@callumforrester what is the desired user-facing behaviour that you want her?

Reply:
I think you were right about this

```shell
curl -X GET ${API}/tasks?status=pending
```

Can that not be done with a pydantic model?

Reply:
relevant issue 
https://github.com/pydantic/pydantic/discussions/4967

Reply:
and also this feature of pydantic
https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.use_enum_values

Reply:
Cheers, I'm convinced that what you've done here is mergable

Reply:
I'm not convinced as now it requires the upper case. I'm leaning to use the enum option I linked

Reply:
Can try it if you like, I'll take a look and let you know what I think, or you can merge as-is

Reply:
ok I'll try later today